### ISSUE TITLE
[A11y] Desktop/Mobile Work Show Page-CPD Parent: 4 image thumbnails

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1957]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-CPD Parent: https://staging.oregondigital.org/concern/generics/df727084j?locale=en

The 4 image thumbnails are throwing up a failed instance error for the following:
image-alt: Ensures <img> elements have alternate text or a role of none or presentation

Path:
.masonry.document.col-xs-6:nth-child(1) > .thumbnail > a > img

Snippet:
`<img src="/downloads/hq37vn58w?file=thumbnail">`

### Suggested Fix

Fix ONE of the following:

- Element does not have an alt attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by CGillen** [comment]
Can't find img's without alt text and masonry style children are out in favor of list (wireframe)
